src/encoded/tests/data/inserts/user.json

```diff
@@ -1155,5 +1155,29 @@
             "ENCORE"
         ],
         "uuid": "64974e05-12df-4733-ae90-b402298b142e"
+    },
+    {
+        "email": "iay@stanford.edu",
+        "first_name": "Ingrid",
+        "groups": [
+            "admin",
+            "verified"
+        ],
+        "job_title": "Data Wrangler",
+        "lab": "/labs/j-michael-cherry/",
+        "last_name": "Youngworth",
+        "schema_version": "8",
+        "status": "current",
+        "submits_for": [
+            "/labs/j-michael-cherry/"
+        ],
+        "viewing_groups": [
+            "community",
+            "ENCODE3",
+            "ENCODE4",
+            "GGR",
+            "REMC",
+            "ENCORE"
+        ],
```

Comment:
This trailing comma will break.  Json files are quite strict on  syntax.  There should be a way to added json syntax to your text editor.  Mine shows errors like this in red when I make them.  Maybe one of the wrangler has this setup in sublime already?    

FYI:  We recommend trailing commas in python files however.

Reply:
Thanks! I will remove it. I haven't finished setting up the linters in Sublime that are recommended, there might be something for JSON in those?